dev/tests/integration/testsuite/Magento/Catalog/_files/url_rewrites.php

```diff
@@ -29,6 +29,7 @@
     ->setCategoryIds([$category1->getId()])
     ->setVisibility(\Magento\Catalog\Model\Product\Visibility::VISIBILITY_BOTH)
     ->setStatus(\Magento\Catalog\Model\Product\Attribute\Source\Status::STATUS_ENABLED)
+    ->setStockData(['use_config_manage_stock' => 1])
```

Comment:
how does the stock data affect url generations?
Looks like problem in our implementation but not in test

Reply:
Can not save product because of new validator.